pysteps/utils/dimension.py

```diff
@@ -206,71 +213,125 @@ def aggregate_fields_space(R, metadata, space_window, ignore_nan=False):
     return R, metadata
 
 
-def aggregate_fields(R, window_size, axis=0, method="mean"):
-    """Aggregate fields.
-    It attemps to aggregate the given R axis in an integer number of sections
+def aggregate_fields(data, window_size, axis=0, method="mean", trim=False):
+    """Aggregate fields along a given direction.
+
+    It attempts to aggregate the given R axis in an integer number of sections
     of length = window_size.
     If such a aggregation is not possible, an error is raised.
```

Comment:
"unless `trim=True`, in which case the axis is trimmed to make it perfectly divisible".